src/components/SwiperQuotes/SwiperQuotes.jsx

```diff
@@ -0,0 +1,61 @@
+"use client";
+import styles from "./SwiperQuotes.module.scss";
+import { Swiper, SwiperSlide } from "swiper/react";
+import { Pagination } from "swiper/modules";
+import "swiper/css/bundle";
+import { Quote } from "../Quote/Quote";
+
+export function SwiperQuotes() {
+  return (
+    <Swiper
+      modules={[Pagination]}
+      spaceBetween={10}
+      slidesPerView={1}
+      pagination={{ clickable: true }}
+      className={styles.swiper}
+    >
+      <SwiperSlide className={styles.slide}>
+        <Quote
```

Comment:
all these quotes should be an array of objects imported. so create a new file called QuoteData.js and export an array of objects. it would reduce this file to like 25 lines. generally, any time somethings repeated more than twice, look to refactor into an array of objects

then here you would do 

`QUOTES.map((quote) => <SwiperSlide><Quote {...quote}/> </SwiperSlide>)`